migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -83,6 +83,13 @@ NFS does not require a defined storage class.
 If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-updating-deprecated-internal-images_troubleshooting-3-4[{product-title} 3 image stream tag] with `podman`.
 * [ ] The target cluster and the replication repository have sufficient storage space.
 * [ ] The xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+* [ ] You have checked the status of the `openshift.io/host.generated` annotation. 
++
+If an application uses an OpenShift route, the route is migrated to the target cluster. The `openshift.io/host.generated` annotation determines whether the host name of the route is updated.
++
+If you wish to update the host name of the route, make sure that `openshift.io/host.generated` is set.  The OpenShift Velero route plugin then strips the source host name from the route and substitutes the target cluster host name.
```

Comment:
"OpenShift Velero route plugin" - unless you plan to define the plugin, you should not mention it.
Use the {product-title} attribute for OpenShift wherever it occurs.